File: Cpp/Math/Sum of first n terms.cpp
//https://www.geeksforgeeks.org/problems/sum-of-first-n-terms5843/1
class Solution {
  public:
    int sumOfSeries(int n) {
        // code here
         int sum=0;
        while(n>0){
            
            sum = sum+(n*n*n);
            n--;
        }
        return sum;
    }
};

File: Python/Math/Sum of first n terms.py
//https://www.geeksforgeeks.org/problems/sum-of-first-n-terms5843/1
class Solution:
    def sumOfSeries(self,n):
        #code here
        total_sum = 0
        while n > 0:
            total_sum += n * n * n
            n -= 1
        return total_sum
